File: src/main/java/ch/uzh/ifi/access/model/dto/ContactDTO.java
package ch.uzh.ifi.access.model.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ContactDTO {
    String name;
    String email;
    String message;
    String topic;

    public String formatContent() {
        return "Name: %s%nEmail: %s%nTopic: %s%nMessage: %s".formatted(name, email, topic, message);
    }
}

File: src/main/java/ch/uzh/ifi/access/model/dto/ContactDTO.kt
package ch.uzh.ifi.access.model.dto

import lombok.AllArgsConstructor
import lombok.Data
import lombok.NoArgsConstructor

@Data
class ContactDTO(
    var name: String? = null,
    var email: String? = null,
    var message: String? = null,
    var topic: String? = null
) {
    fun formatContent(): String {
        return "Name: $name\nEmail: $email\nTopic: $topic\nMessage: $message"
    }
}
